packages/replay-next/src/hooks/useTooltip.tsx

```diff
@@ -8,6 +17,129 @@ type TooltipPosition = "above" | "below" | "left-of" | "right-of";
 export const MARGIN_LARGE = 8;
 export const MARGIN_SMALL = 4;
 
+// A copy-pasted version of the other `useTooltip` hook,
+// but exposing `setShowTooltip` for control from the parent component
+// instead of controlling display itself on hover.
+export function usePositionedTooltip({
```

Comment:
This seems like an awkward mix of default and named exports. Why is this in the same file?

Reply:
This and the other comment are the same reason: I was going for the simplest thing that would work, you said it made sense to fork it, and given that we might re-combine them later it didn't seem worth putting this in another file.

Reply:
Not knowing what you had in mind, forking seemed reasonable. Seeing this proposed implementation though, it seems like overkill. I dunno 😄